mobile/src/modules/auth/screens/AccountDetailsScreen.tsx

```diff
@@ -0,0 +1,70 @@
+import { Account } from '@burstjs/core';
+import React from 'react';
+import { Text } from 'react-native';
+import { NavigationInjectedProps, withNavigation } from 'react-navigation';
+import { connect } from 'react-redux';
+import { AccountDetailsHeaderTitle } from '../../../core/components/header/AccountDetailsHeaderTitle';
+import { BackHeaderButton } from '../../../core/components/header/BackHeaderButton';
+import { i18n } from '../../../core/i18n';
+import { InjectedReduxProps } from '../../../core/interfaces';
+import { FullHeightView } from '../../../core/layout/FullHeightView';
+import { Screen } from '../../../core/layout/Screen';
+import { routes } from '../../../core/navigation/routes';
+import { ApplicationState } from '../../../core/store/initialState';
+import { auth } from '../translations';
+
+interface Props extends InjectedReduxProps {
+  account: Account
+}
+
+type TProps = NavigationInjectedProps & Props;
+
+interface State {
+}
+
+class AccountDetails extends React.PureComponent<TProps, State> {
+  state = {
+  };
+
+  static navigationOptions = ({ navigation }: NavigationInjectedProps) => {
+    const { params = {} } = navigation.state;
+
+    return {
+      headerLeft:  <BackHeaderButton onPress={params.handleBackPress} />,
+      headerTitle: (<AccountDetailsHeaderTitle>
```

Comment:
This will all change to more closely reflect this comp:
https://media.discordapp.net/attachments/535959194851999765/582762659389374484/unknown.png?width=321&height=570